tests/unittests/ImporterTestUtils.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "glow/Base/Tensor.h"
+#include "glow/Graph/Graph.h"
+
+namespace glow {
+
+/// Generate the input image in NCHW format used
+/// for convolution op input.
+void getImageData(Tensor *result, size_t n, size_t c, size_t h, size_t w) {
```

Comment:
The term image here doesn't help to understand what is happening here.
In the sense that really we assign random number here and it is hard to claim that we generate an image, let alone one that makes sense with the NCHW format :).
If we want to deduce anything from the generated data, we need to document what it does.

Maybe something like:
```
Populate \p result with increasing number following the NCHW format
E.g., for N(1) x C(3) x H(3) x W(3)
                W
              <--->
     ^    +----+----+---+
  C /     | 18 | 19 | 20|  ^
   / +---+----+---+ | 23|  | H
  v  | 9 | 10 | 11| | 26|  v
+---+---+---+ | 14|-+---+
| 0 | 1 | 2 | | 17|
| 3 | 4 | 5 |-+---+
| 6 | 7 | 8 |
+---+---+---+

```

Note: Double check what we actually output the number in that order :P